documentation/templatetags/documentation_extras.py

```diff
@@ -8,12 +8,8 @@
 
 @register.filter
 def get_absolute_url_for_host(map, host):
-    if host == 'curriculum.code.org' or host == 'www.codecurricula.com' or host == 'localhost:8000' or host == 'testserver':
-        return '/documentation/%s/' % map.slug
-    elif host == 'studio.code.org':
+    if host == 'testserver' or host == 'localhost:8000':
         return '/docs/%s/' % map.slug
-    elif host == 'docs.code.org':
-        return '/%s/' % map.slug
     else:
         logger.info("no known host %s" % host)
-        return '/%s/' % map.slug
\ No newline at end of file
+        return '/docs/%s/' % map.slug
```

Comment:
since this URL pattern matches the above, there should be no need for this conditional any more